hkm/static/hkm/js/cookie-consent.js

```diff
@@ -0,0 +1,73 @@
+(function($) {
+    var cookieName = "hki_consent"
+    var cookie = $.cookie(cookieName)
+    var _paq = window._paq = window._paq || [];
+
+    const $cookieConsentContainer =  $("#cookie-consent"), 
+    $readMoreBtn = $("#cookie-consent__readmore-btn"),
+    $cookieContents = $("#cookie-consent__content"),
+    $acceptBtn = $("#consent_accept"),
+    $rejectBtn = $("#consent_reject"),
+    $consentLink = $("#cookie-consent-link");
+
+    // if rejected
+    if(cookie && cookie === 0) {
+        _paq.push(['forgetCookieConsentGiven']);
+        _paq.push(['requireCookieConsent']);
+    }
+
+    // if accepted
+    if(cookie && cookie === 1) {
```

Comment:
"Else if" would make this clearer, because there is no return statement inside an if-block. Only 1 of the blocks is matching. I would probably split the function to multiple (sub) functions too, but it may be an overkill.